Pear/Profile/ProfileViewController.swift

```diff
@@ -13,13 +13,15 @@ class ProfileViewController: UIViewController {
 
     // MARK: - Private View Vars
     private let backButton = UIButton()
-    private var user: UserV2?
+    private var currentUser: UserV2
+    private var otherUser: UserV2?
```

Comment:
Having this optional variable and also having `otherUserId` is redundant. What if you made an enum with two states? 

```
private enum ProfileViewControllerUser {
    case userId(String)
    case user(UserV2)
}
```

Maybe this is unnecessary. Let me know your thoughts.